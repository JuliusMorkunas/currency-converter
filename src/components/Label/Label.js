import React from 'react';
import s from './Label.module.scss';

const Label = ({ children, ...rest }) => {
  return (
    <label {...rest} className={s.Label}>
      {children}:
    </label>
  );
};

export default Label;
