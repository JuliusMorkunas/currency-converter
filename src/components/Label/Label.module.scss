@import '../../variables';

.Label {
  font-size: 11px;
  text-transform: uppercase;
  color: $c-grey;
}
