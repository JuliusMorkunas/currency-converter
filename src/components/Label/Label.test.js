import React from 'react';
import { render } from '@testing-library/react';
import Label from './Label';

it('Label should match snapshot', () => {
  const { asFragment } = render(<Label />);
  expect(asFragment()).toMatchSnapshot();
});
