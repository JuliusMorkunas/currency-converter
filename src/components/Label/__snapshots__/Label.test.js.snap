// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should match snapshot 1`] = `
<DocumentFragment>
  <label
    class="Label"
  >
    :
  </label>
</DocumentFragment>
`;
